mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/entity/EntityHelperSchemaValidationMethodGenerator.java

```diff
@@ -109,18 +109,25 @@ public EntityHelperSchemaValidationMethodGenerator(
 
     generateValidationChecks(methodBuilder);
 
-    // Throw if there is not keyspace.table for defined entity
-    CodeBlock missingKeyspaceTableExceptionMessage =
-        CodeBlock.of(
-            "String.format(\"There is no ks.table: %s.%s for the entity class: %s\", keyspaceId, tableId, entityClassName)");
-    methodBuilder.beginControlFlow("else");
-    methodBuilder.addStatement(
-        "throw new $1T($2L)", IllegalArgumentException.class, missingKeyspaceTableExceptionMessage);
-    methodBuilder.endControlFlow();
+    logMissingMetadata(methodBuilder);
 
     return Optional.of(methodBuilder.build());
   }
 
+  private void logMissingMetadata(MethodSpec.Builder methodBuilder) {
+    methodBuilder.addComment(
```

Comment:
There were some failures of https://jenkins-drivers.build.dsinternal.org/job/datastax.java-driver.java2263_rebase.default/lastCompletedBuild/testReport/ `DefaultKeyspaceId`, it reported that the table is not present (it fails only on jenkins) I suspect that there is some kind of race condition in metadata and this is related to your comment:
>keyspace name present but metadata is out of date

I don't know how can we alleviate that problem - maybe we could refresh the schema explicitly?
If not, I think the viable solution is to `log.warning`:
`warn if there are is keyspace.table for defined entity - it means that table is missing, or schema it out of date.`
instead of throwing an exception and fail fast if we cannot be sure if the schema was out-dated.